tools/downloader/README.md

```diff
@@ -11,6 +11,9 @@ based on configuration files in the models' directories.
 * `converter.py` (model converter) converts the models that are not in the
   Inference Engine IR format into that format using Model Optimizer.
 
+* `quantizer.py` (model quantizer) quantizes full-precision models in the IR
+  format into low-precision versions using Post-Training Optimization Toolkit.
```

Comment:
What do you think about adding link to toolkit?

Reply:
How would I link to it?

Reply:
For example, link to documentation. I don't insist on it, it is just a suggestion.

Reply:
Hmm... it wouldn't be a bad idea, but the problem is that URL for the the POT documentation is currently "https://docs.openvinotoolkit.org/latest/_README.html", which seems rather unstable to me. I don't want to link to something like that might end up pointing to a completely different page later.

Reply:
Ok, I got it.